posix.mak

```diff
@@ -283,7 +283,7 @@ install :
 
 .PHONY : unittest
 ifeq (1,$(BUILD_WAS_SPECIFIED))
-unittest : $(addsuffix .d,$(addprefix unittest/,$(D_MODULES)))
+unittest : $(addsuffix .run,$(addprefix unittest/,$(D_MODULES)))
```

Comment:
Yep, just another of those lovely misfeatures of make that everybody just loves... :-P